geanydoc/src/Makefile.am

```diff
@@ -7,7 +7,7 @@ geanydoc_la_SOURCES = \
 	geanydoc.c \
 	geanydoc.h
 
-geanydoc_la_CFLAGS = $(AM_CFLAGS)
+geanydoc_la_CPPFLAGS = $(AM_CPPFLAGS) -DG_LOG_DOMAIN=\"GeanyDoc\"
```

Comment:
I replaced `geanydoc_la_CFLAGS` by accident. But I think this line was useless already before because $(AM_CFLAGS) is probably the default. If I'm wrong, I'd repush without that line removed.